Source/WTF/wtf/Dominators.h

```diff
@@ -25,9 +25,11 @@
 
 #pragma once
 
+#include <wtf/BitSet.h>
 #include <wtf/CommaPrinter.h>
 #include <wtf/FastBitVector.h>
 #include <wtf/GraphNodeWorklist.h>
+#include <wtf/MonotonicTime.h>
```

Comment:
Do we need this?

Reply:
Ah, no, good catch - that was just for measuring the time we spend computing dominators. I'll remove it.